performant-software/Annotation-Studio
# Fix JS Tinymce error when swtiching users

## What does this PR do?
1. Remove the Javascript Tinymce error that occurs when an admin is switching users on the document show

## How to test?
1. Sign in as an admin
2. Navigate to a document
3. Open js console
4. Switch to a different user using the user switch menu at bottom of the right hand filter sidebar.
5. Confirm that there is no tinymce js error message in the javascript console
